Sources/THFeedView/LibraryComponents/Layouts/THLayouts.swift

```diff
@@ -58,4 +58,28 @@ public class THFeedLayouts {
                 
         return section
     }
+    
+    public class func carouseLayout() -> NSCollectionLayoutSection {
```

Comment:
Esta mal escrito carousel

```suggestion
    public class func carouselLayout() -> NSCollectionLayoutSection {
```